src/cohpsk.c

```diff
@@ -771,7 +771,24 @@ void fdm_downconvert_coh(COMP rx_baseband[COHPSK_NC][COHPSK_M+COHPSK_M/P], int N
     }
 }
 
+/* Determine if we can use vector ops below. */
+#if __GNUC__ > 4 || \
```

Comment:
@drowe67, I think this is correct to ensure that the vector calls are used only for supporting compilers (e.g. [this](https://stackoverflow.com/questions/1771945/c-how-to-access-elements-of-vector-using-gcc-sse-vector-extension) has a comment indicating that vector subscript addressing was introduced in GCC 4.6), but we may want to double-check to be sure.